api/envoy/service/discovery/v3/discovery.proto

```diff
@@ -272,4 +273,13 @@ message Resource {
 
   // The resource being tracked.
   google.protobuf.Any resource = 2;
+
+  // Time-to-live value for the resource. For each resource, a timer is started. The timer is reset each time the
+  // resource is received with a new TTL. If the resource is received with no TTL set, the timer is removed for the
+  // resource. Upon expiration of the timer, the configuration for the resource will be removed.
```

Comment:
Should we add a comment here documenting the fact that TTLs can be updated/removed by re-sending the `Resource` with the same version but without populating the `resource` field itself?